Source/WebCore/Modules/mediasource/MediaSource.cpp

```diff
@@ -126,6 +219,24 @@ MediaSource::~MediaSource()
     ASSERT(isClosed());
 }
 
+#if !RELEASE_LOG_DISABLED
+Ref<Logger> MediaSource::logger(ScriptExecutionContext& context)
+{
+    if (RefPtr document = dynamicDowncast<Document>(context))
+        return document->logger();
```

Comment:
Hm, this is something I didn't consider: moving to a worker means losing logging. :-/

Reply:
fixed